modules/gatk_rnaseq/1.0/config/default.yaml

```diff
@@ -0,0 +1,66 @@
+lcr-modules:
+    
+    gatk_rnaseq:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            sample_bai: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            gatk_variant_calling:
+                min_conf_thres: 20.0
+            gatk_variant_filtration:
+                window: 35 # window size between SNPs in cluster
+                cluster_size: 3 # at least 3 SNPs in cluster
+                # hard filtering (filters OUT) based on metrics: 
+                    # FS (FisherStrand): Phred-scale probability that there is a strand bias from a Fisher's test. (default FS > 30.0)
+                    # QD (QualByDepth): variant confidence/unfiltered depth (normalizes variant quality to avoid inflation from deep coverage) (default QD < 2.0)
+                    # DP (depth): minimum depth (default DP < 5.0)
+                filter_expression: "-filter-expression \"FS > 30.0\" -filter-name FS -filter-expression \"QD < 2.0\" -filter-name QD -filter-expression \"DP < 5.0\" -filter-name DP"
+            gatk_rnaseq_filter_passed: 
+                params: "-f '.,PASS' "
+                # Can be modified to filter on additional criteria using bcftools view syntax
+                # For example, to remove all variants with -log10(POPAF) > 4.0: 
+                #"-f '.,PASS' -i 'INFO/POPAF > 4'"
+
+
+        conda_envs:
+            gatk_rnaseq: "{MODSDIR}/envs/gatk_rnaseq.yaml"
+            bcftools: "{MODSDIR}/envs/bcftools-1.10.2.yaml"
+            
+        threads:
+            gatk_splitntrim: 12
```

Comment:
It is possible to combine these keys and reuse them across rules. In other words, if several rules need the same number of threads, they can refer to the same key in config. Same can be applied to resources as well. I think this reduces the number of keys to specify/adjust if needed, and reduces complexity of the config. What do you think?

Reply:
You are right, let's keep the more detailed and informative names